dashboard/src/views/SystemSpec.vue

```diff
@@ -111,7 +114,11 @@ export default class SystemSpec extends Vue {
   data() {
     return {
       system: this.system,
-      modelPresetOptions: ["pvsyst", "pvwatts"],
+      modelPresetOptions: {
+        pvsyst: "pvsyst-like",
+        pvwatts: "pvwatts",
+        sam: "SAM"
+      },
       model: "pvsyst",
```

Comment:
I think we probably want to set SAM as the default since it provides searchable modules + inverters for new users.

Reply:
Thoughts @wholmgren @cwhanse ?